tests/functional/fuzzy/test_fuzzy.py

```diff
@@ -98,14 +112,195 @@ def test_fuzz_config():
+
+    # Create trace files
+    iotrace.start_tracing()
+    iotrace.stop_tracing()
+    trace_repo_path = iotrace.parse_json(iotrace.get_trace_repository_path())[0]['path']
+    trace_path = trace_repo_path + "/" + iotrace.get_latest_trace_path()
+    summary_path = f'{trace_path}/octf.summary'
+    copied_summary_path = f'{repo_path}/rootfs/var/lib/octf/trace/' + \
+                            f'{iotrace.get_latest_trace_path()}/octf.summary'
+
+    # Create patch file for redirecting fuzzed stdin to trace file
+    new_patch_path: str = f'{iotrace.working_dir}/redirect_to_summary.patch'
+    create_patch_redirect_fuzz_to_file(f'{copied_summary_path}', new_patch_path)
+
+    # Install iotrace locally with AFL support and redirect patch
+    install_iotrace_with_afl_support(new_patch_path)
+
+    # Copy trace files to local installation of iotrace
+    TestRun.executor.run_expect_success(f'cp -r {trace_repo_path}/kernel '
+                                        f'{repo_path}/rootfs/var/lib/octf/trace/')
+
+    # Instruct the system to output coredumps as files instead of sending them
+    # to a specific crash handler app
+    TestRun.LOGGER.info('Setting up system for fuzzing')
+    TestRun.executor.run_expect_success('echo core > /proc/sys/kernel/core_pattern')
+    TestRun.executor.run_expect_success(f'cd {repo_path} && mkdir -p afl-i afl-o')
+
+    # Add input seed which shall be mutated
+    TestRun.executor.run_expect_success(f'cd {repo_path} ' +
+                                        '&& echo "{}" > afl-i/case0')
+
+    TestRun.LOGGER.info(f'Starting fuzzing {summary_path} This should take ' +
+                        str(fuzzing_time_seconds / 60) + ' minutes')
+    TestRun.executor.run(f'cd {repo_path} && ./tests/functional/fuzzy/fuzz.sh '
+                         '"rootfs/bin/iotrace --get-trace-statistics -p '
+                         f'{iotrace.get_latest_trace_path()}" --one-job')
+    output = wait_for_completion(fuzzing_time_seconds, repo_path)
+    TestRun.executor.run(f'cd {repo_path} && ./tests/functional/fuzzy/fuzz.sh clean')
+    detect_crashes(output.stdout)
+
+
+def test_fuzz_procfs():
+    # For simplicity we start one non-instrumented iotrace instance
+    # and fuzz proc files using afl with -n flag (dumb fuzzing)
+    fuzzing_time_seconds = 15 * 60
+    iotrace: IotracePlugin = TestRun.plugins['iotrace']
+
+    repo_path: str = f"{iotrace.working_dir}/slit-afl"
+
+    # Make sure AFL is installed
+    if not is_afl_installed():
+        install_afl()
+
+    # Instruct the system to output coredumps as files instead of sending them
+    # to a specific crash handler app
+    TestRun.LOGGER.info('Setting up system for fuzzing')
+    TestRun.executor.run_expect_success('echo core > /proc/sys/kernel/core_pattern')
+    TestRun.executor.run_expect_success(f'cd {repo_path} && mkdir -p afl-i afl-o')
+
+    # Procfs files and their initial seed
+    fuzzed_files = {"/proc/iotrace/add_device": "/dev/sdb",
```

Comment:
Shouldn't /dev/sdb be taken from some test argument/parameter?

Reply:
/dev/sdb here is a seed value for fuzzing. This means that every bit of this string will be mutated, so the specific initiakl value of it is not that important. It might as well be just "/dev/", but I think full path will successfully mutate more quickly.